device/transport/amqp/src/amqp_device_method_client.ts

```diff
@@ -88,7 +88,8 @@ export class AmqpDeviceMethodClient extends EventEmitter {
                     properties: {
                       'com.microsoft:api-version': endpoint.apiVersion,
                       'com.microsoft:channel-correlation-id': credentials.deviceId
-                    }
+                    },
+                    rcvSettleMode: 0
```

Comment:
>0 [](start = 35, length = 1)

grumble.  I would prefer a named value here, but....whatever.  maybe add a comment saying that it's settleFirst or whatever you want to call it.